next-lesson/schedule.md

```diff
@@ -0,0 +1,9 @@
+<div style="background: red; color: black;">FIXME</div>
+
+Please delete the contents of this file and fill it with your own
+
+## Expected contents
+
+The times and what you will be teaching at each day in a time-table format.
```

Comment:
It would be nice to prefill it with the table with an example 2-day schedule.

Reply:
good idea, it is added.